_posts/2018-12-19-bazel-0.21.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: posts
+title: Bazel 0.21
+authors:
+  - laurentlb
+---
+
+We've just released [Bazel 0.21](https://github.com/bazelbuild/bazel/releases/tag/0.21.0)!
+
+We also revamped our Android with Bazel tutorial. Check it out [here](https://docs.bazel.build/versions/master/tutorial/android-app.html).
```

Comment:
What's the reason for having moved this out of the Android section?

Reply:
I wanted to highlight it (also, it's not directly a change to Bazel).

Reply:
That implies to me that this announcement is more important than the others, and I don't see why.